bin/impl/fluo_deploy/main.py

```diff
@@ -93,6 +93,12 @@ def launch_cluster(conn, config):
       exit('ERROR - Image not found for instance type: '+instance_type)
 
     bdm = BlockDeviceMapping()
+    if config.ebs_root_size() > 0 :
+      bdt = BlockDeviceType()
```

Comment:
I just tried this code w/ m3.xlarge and the instances would not start.  I suspect it may be this code.   I set the option to 0 and things started